client/server/bundler/sections-loader.js

```diff
@@ -67,6 +78,10 @@ const loader = function() {
 			console.warn( `[sections-loader] Available sections are:` );
 			printSectionsAndPaths( allSections );
 		}
+	} else {
+		console.log(
+			`[sections-loader] Built ${ sections.map( section => section.name ).join( ', ' ) } sections`
```

Comment:
Would be really cool if this supports a singular case. Also, is this something we could reuse `printSectionsAndPaths` for?

Reply:
:+1: yeah I think that could work. I was mostly using it for my testing but I think this makes a lot of sense.